src/main/webapp/js/common.es6

```diff
@@ -538,24 +538,61 @@ function toggleSort(divElement, comparator) {
 
     const $selectedDivElement = $(divElement);
 
-    if ($selectedDivElement.attr('class') === 'button-sort-none') {
+    if ($selectedDivElement.attr('class') === 'button-sort-none toggle-sort') {
         sortTable(divElement, colIdx, comparator, true, row);
-        $selectedDivElement.parent().find('.button-sort-ascending').attr('class', 'button-sort-none');
-        $selectedDivElement.parent().find('.button-sort-descending').attr('class', 'button-sort-none');
+        $selectedDivElement.parent().find('.button-sort-ascending').attr('class', 'button-sort-none toggle-sort');
+        $selectedDivElement.parent().find('.button-sort-descending').attr('class', 'button-sort-none toggle-sort');
```

Comment:
Yep, this is was just a temporary hack for me to figure out why tableSort was failing. I'll clean up this mess at the end before calling for review.